tensorboard/plugins/profile/pod_viewer/pod_viewer_dashboard/pod-viewer-dashboard.ts

```diff
@@ -0,0 +1,232 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+namespace pod_viewer_dashboard {
+
+Polymer({
+  is: 'pod-viewer-dashboard',
+  properties: {
+    _data: {
+      type: Object,
+      notify: true,
+    },
+    podStats: {
+      type: Object,
+      notify: true,
+      observer: 'podStatsChanged_',
+    },
+    stepStats: {
+      type: Array,
+      value: null,
+      notify: true,
+    },
+    channelDb: {
+      type: Array,
+      value: null,
+      notify: true,
+    },
+    allReduceDb: {
+      type: Array,
+      value: null,
+      notify: true,
+    },
+    stepBreakdownEle: {
+      type: Array,
+      notify: true,
+    },
+    channelEle: {
+      type: Array,
+      notify: true,
+    },
+    allReduceEle: {
+      type: Array,
+      notify: true,
+    },
+    stepBreakdownFunc: {
+      type: Object,
+      notify: true,
+    },
+    channelFunc: {
+      type: Object,
+      notify: true,
+    },
+    allReduceFunc: {
+      type: Object,
+      notify: true,
+    },
+    runEnvironment: {
+      type: Object,
+      notify: true,
+    },
+    curStepId: {
+      type: Number,
+      value: 0,
+      observer: 'stepChanged_',
+    },
+    maxStepId: {
+      type: Number,
+    },
+    stepNum: {
+      type: Number,
+      computed: 'getStepNum(podStats)',
+    },
+    selectedChipId: {
+      type: Number,
+      value: -1,
+      notify: true,
+    },
+    selectedChannel: {
+      type: Array,
+      notify: true,
+      observer: 'selectedChannelChanged_',
+    },
+    activeBarChartEle: {
+      type: Object,
+      notify: true,
+      observer: 'activeBarChartEleChanged_',
+    },
+    hloInfoMap: {
+      type: Object,
+      notify: true,
+    },
+    active: {
```

Comment:
property `active` is ambiguous: without reading this line, I assumed it was a boolean. Perhaps the word `active` is not descriptive enough. What is this property about? I am hoping that it is self-descriptive.

Reply:
Done.